Wrappers.h

```diff
@@ -0,0 +1,40 @@
+
+
+#pragma once
+
+#include "Car.h"
+#include "Motorbike.h"
+#include "Lorry.h"
+#include "Motorway.h"
+
```

Comment:
You aren't using any of these in this header file.  but you are in the cpp.  Include only what you use.
- [ ] revise.

Reply:
I'm having difficulty understanding this one.  I moved the includes to the cpp and instead forward declared the structs, but I have to include Motorway.h otherwise I get 

error: member access into incomplete type 'Motorway'

errors.  I'm not sure why?

Reply:
Look at which file the `member access into incomplete type ` is being triggered.  it's not being triggered in the Wrappers file...